ui/src/app/components/uploads/list.component.ts

```diff
@@ -141,6 +146,45 @@ export class ListComponent extends BaseComponent implements OnInit {
     }
   }
 
+  checkForLinkedTestCases(testData?,id?) {
+    let testCases: InfiniteScrollableDataSource;
+    let query = "workspaceVersionId:" + this.versionId + ",testData:" + encodeURI(testData?.join("#"))
+    query = this.byPassSpecialCharacters(query);
+    testCases = new InfiniteScrollableDataSource(this.testCaseService,query);
+
+    waitTillRequestResponds();
+    let _this = this;
+
+    function waitTillRequestResponds() {
+      if (testCases.isFetching)
+        setTimeout(() => waitTillRequestResponds(), 100);
+      else {
+        if (testCases.isEmpty)
+          _this.openDeleteDialog(id);
+        else
+          _this.openLinkedTestCasesDialog(testCases);
+      }
+    }
+  }
+  private openLinkedTestCasesDialog(list) {
```

Comment:
@Raksha-testsigma Needs a little formatting. In some cases, there is no line gap between two functions. In some other cases, there are quite a few. Use uniform line gaps